echo.go

```diff
@@ -500,6 +500,7 @@ func (common) static(prefix, root string, get func(string, HandlerFunc, ...Middl
 		}
 		return c.File(name)
 	}
+	get(prefix, h)
```

Comment:
Then dealing with this method we need to consider where is is called.  Is it  `e.Static()` or `group.Static()`

* `e.Static()` is on server root level i.e.  `http://server/` URLs
* `group.Static` is on group level which means in addition to this method prefix we have group path as prefix in url. i.e. `http://server/group/`

and regarding to `prefix` we should consider it ends with slash or not
i.e. prefixes:
* ``
* `/`
* `/assets`
* `/assets/`

now lets add two possibilities here: if group path is `/user` or `/user/` what routes we will end up with?

Reply:
something like that

```go
	// prefix does not end with slash
	if prefix != "" && prefix[len(prefix) - 1] != '/' {
		get(prefix, h) // i.e. prefix=/assets and URL `/assets` must give directory index
		return get(prefix+"/*", h) // and asterisk to match anything in that directory
	}
	if prefix != "" {
		get(prefix[:len(prefix) - 1], h) // i.e. prefix=/assets/ then add route `/assets` for directory index
	}
	// i.e. prefix="" (empty) and URL `http://server` or `http://server/` must give directory index
	// i.e. prefix=/assets/ and URL `/assets/` must give directory index, `/assets/*` serve files
	return get(prefix+"*", h)
```